scripts/init_db.sql

```diff
@@ -0,0 +1,95 @@
+CREATE TABLE books (
+    id SERIAL PRIMARY KEY,
+    ISBN TEXT UNIQUE,
+    title TEXT NOT NULL,
+    synopsis TEXT NOT NULL,
+    publisher TEXT NOT NULL,
+    publishing_date TEXT NOT NULL,
+    edition TEXT NOT NULL
```

Comment:
Creo que ya hablamos de esto, pero ¿crees que ser tan preciso en la descripción de libro sirve para algo? ¿Vas a usar la edición en la lógica de negocio?
El resto puede servir para identificar de forma única, pero no tengo muy claro que este en concreto sirva de algo.

Reply:
La edición se puede utilizar para identificar el libro. Hay algunos casos en los que nuevas ediciones han cambiado términos o expresiones originales por otras más políticamente correctas o aceptadas en la actualidad.

Ya que esto puede afectar ligeramente al estilo, creo que es relevante.

Una noticia de ejemplo: https://www.theguardian.com/books/2023/feb/27/james-bond-novels-to-be-reissued-with-racial-references-removed